src/inc/corjit.h

```diff
@@ -301,7 +301,6 @@ enum CorJitFuncKind
     CORJIT_FUNC_FILTER         // a funclet associated with an EH filter
 };
 
-#if !defined(FEATURE_USE_ASM_GC_WRITE_BARRIERS) && defined(FEATURE_COUNT_GC_WRITE_BARRIERS)
 // We have a performance-investigation mode (defined by the FEATURE settings above) in which the
```

Comment:
Done. We should consider enabling this mode with a JIT flag instead of preprocessor syumbols.